package-lock.json

```
@@ -455,7 +455,7 @@
         },
         "minimist": {
           "version": "1.2.0",
-          "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
+          "resolved": "http://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```

Revert the change please, it should be `https` here and in other places
```suggestion
          "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```